actions/unit.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 # DESCRIPTION: Execute unit tests
 
-phpdbg -qrr bin/phpunit --coverage-clover=./build/coverage.xml --coverage-html=./build/html-coverage
+bin/phpunit --coverage-clover=./build/coverage.xml --coverage-html=./build/html-coverage
```

Comment:
I think that the settings for pcov are missing here.

Reply:
thats removed in favor of the dockerfiles in the project. If you want coverage either execute through docker or configure yourself. I thought it would be good to not assume that pcov is available (eg. it is not on my machine only in containers)